en/streaming-integrator/docs/examples/handling-requests-with-errors.md

```diff
@@ -326,4 +352,104 @@ To manage the error in the Error Store Explorer, follow the procedure below:
+    );
+    ```
+#### Step 7.2: Deploy the StoreRdbmsError siddhi application
+
+In this step, let's start the service at `http://localhost:8006/insertStream` via the `StoreRdbmsError` Siddhi 
+application
+
+In the **Siddhi Apps to Deploy** section, select the check box for the **StoreRdbmsError.siddhi** application. In the 
+**Servers** section, select the check box for the server you added. Then click **Deploy**.
+
+   ![Select StoreRdbmsError Siddhi Application and Server](../../images/handling-requests-with-errors/select-StoreRdbmsError-app-and-server.png)
+
+The following log is displayed in the Streaming Integrator console.
+    ```
+    INFO {org.wso2.carbon.streaming.integrator.core.internal.StreamProcessorService} - Siddhi App StoreRdbmsError deployed successfully
+    ```
+
+#### Step 7.2: Publish an event into StoreRdbmsError Siddhi application
+
+Send an event to the insertStream stream of the StoreRdbmsError Siddhi application by issuing the following CURL 
+command.
+
+    `curl --location --request POST 'http://localhost:8006/insertStream' --header 'Content-Type: application/json' --data-raw ' { "event": { "batchID": 1, "amount": 45.6, "factoryID": 102 } }'`
+
+Check the contents of `SweetProductionTable` to see the published event successfully stored in the table
+
+    ```
+    mysql> select * from SweetProductionTable;
+    +---------+--------+-----------+
+    | batchID | amount | factoryID |
+    +---------+--------+-----------+
+    |       1 |   45.6 |       102 |
+    +---------+--------+-----------+
+    1 row in set (0.00 sec)
+    ```
+
+Use the same curl command as above to publish another event. However, this time our event will fail since the batchID 
+column which is defined as the primary key has a duplicate value of 1 in the second message. You will see the following 
+log in the wso2si terminal.
+
+    ```
+    [2020-11-03 11:48:28,951] ERROR {io.siddhi.core.table.Table} - Error on 'StoreRdbmsError' while performing add for events  at 'SweetProductionTable'. Events saved 'EventChunk{first=StreamEvent{ timestamp=1604384308930, beforeWindowData=null, onAfterWindowData=null, outputData=[1, 45.6, 102], type=CURRENT, next=null}}'
+    ```
+#### Step 7.3: Manage the error in the Error Store Explorer
+To manage the error in the Error Store Explorer, follow the procedure below:
+    
+1. To open the Error Store Explorer, open Streaming Integrator Tooling, click **Tools** and then click 
+**Error Store Explorer**.
+
+    ![Access Error Store](../../images/handling-requests-with-errors/access-error-store-explorer.png)
+    
+2. In the **Siddhi app** field, select **StoreRdbmsError** Siddhi application and then click **Fetch**.
+    
+    As a result, an error is displayed as follows.
+    
+    ![Error Store Explorer](../../images/handling-requests-with-errors/error-store-explorer-with-StoreRdbmsError.png)
+    
+    This indicates that the event was dropped because the end point was not available.
```

Comment:
Let's correct this to `... dropped due to a constraint violation`

Reply:
Fixed in a5ef3f0